src/System.Runtime.Serialization.Json/tests/ReflectionOnly/Resources/System.Runtime.Serialization.Json.ReflectionOnly.Tests.rd.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Directives xmlns="http://schemas.microsoft.com/netfx/2013/01/metadata">
+  <Library Name="*System.Private.DataContractSerialization*">
+    <Assembly Name="System.Private.DataContractSerialization" Dynamic="Required All">
```

Comment:
It's probably due to [this line](https://github.com/tijoytom/corefx/blob/71799e2599f17d24919c6f93da9c97310d7eb2c2/src/System.Runtime.Serialization.Json/tests/DataContractJsonSerializer.cs#L24), which sets the reflection mode to run the tests using reflection based serialization. If that' the case, I think we can modify the rd.xml to keep the metadata for `DataContractSerializer.set_Option` only.